tests/Parser/SalesPricesTest.php

```diff
@@ -41,27 +56,59 @@ public function testParse($data, $expectedResult)
         $this->assertSame($expectedResult, $salesPricesMock->getResults());
     }
 
-    public function getRRPProvider()
+    public function providerGetDefaultPrice()
+    {
+        return array(
+            array(
+                array(),
+                false
+            ),
+            array(
+                array('default' => array(2, 4)),
+                2
+            )
+        );
+    }
+
+    /**
+     * @dataProvider providerGetDefaultPrice
+     */
+    public function testGetDefaultPrice($parsedData, $expectedResult)
+    {
+        $salesPricesMock = $this->getMockBuilder('\Findologic\Plentymarkets\Parser\SalesPrices')
+            ->disableOriginalConstructor()
+            ->setMethods(null)
+            ->getMock();
+
+        $salesPricesMock->setResults($parsedData);
+        $this->assertSame($expectedResult, $salesPricesMock->getDefaultPrice());
+    }
+
+    public function providerGetDefaultRrp()
     {
         return array(
-            // No prices with RRP type, results should be empty
-            array(array('1' => 'default'), array()),
-            // Two price has RRP type, results should contain ids of those prices
-            array(array('1' => 'default', '2' => SalesPrices::RRP_TYPE, '3' => SalesPrices::RRP_TYPE), array(2, 3))
+            array(
+                array(),
+                false
+            ),
+            array(
+                array('rrp' => array(1, 2)),
+                1
+            )
         );
     }
 
     /**
-     * @dataProvider getRRPProvider
+     * @dataProvider providerGetDefaultRrp
      */
-    public function testGetRRPProvider($salesPrices, $expectedResult)
+    public function testGetDefaultRrp($parsedData, $expectedResult)
```

Comment:
- [ ] Add the argument definitions to this doc block.

- [ ] Add the type hint for `$parsedData` e.g. `array $parsedData`.